src/steps/ShowAddress/index.tsx

```diff
@@ -88,30 +102,93 @@ const ShowAddress: React.FC<Props> = () => {
     };
   }, []);
 
+  useEffect(() => {
+    if (!state.depositAddress.data?.address) {
+      return;
+    }
+
+    switch (state.network?.networkCode) {
+      case 'bitcoin':
+      case 'litecoin':
+      case 'bitcoincash':
+      case 'doge':
+        const bip21 = encode(
+          state.depositAddress.data?.address,
+          {
+            amount: state.requiredAmount,
+          },
+          state.asset?.networkCode
+        );
+
+        return setQrValue(bip21);
+      case 'ethereum':
+      case 'goerli':
+      case 'polygon':
+      case 'optimism':
+      case 'arbitrum':
+      case 'smartchain':
+      case 'avalanchec':
+      case 'fantom':
+      case 'cronos':
+      case 'palm':
+      case 'ronin':
+        // TODO - support ERC20
```

Comment:
Couldn't seem to get erc20 support working, though it should technically. I've left a todo and created a ticket